rules/manager/manager.go

```diff
@@ -229,9 +236,11 @@ func (m *ruleManager) runIteration(results chan<- *extraction.Result) {
 					Timestamp: s.Timestamp,
 				}
 			}
-			m.results <- &extraction.Result{
-				Samples: samples,
-				Err:     err,
+
+			if err != nil {
+				evalFailures.Inc()
```

Comment:
I'd tend towards glog.Info. It technically should be Warning, but I expect it to get very spammy for lots of users due to improper rules.